libs/network/include/network/details/idle_work_store.hpp

```diff
@@ -92,7 +92,13 @@ class IdleWorkStore
 
     if (mutex_.try_lock())
     {
-      is_due = Clock::now() >= (last_run_ + interval_);
+      try
+      {
+        is_due = Clock::now() >= (last_run_ + interval_);
+      }
+      catch (...)
+      {
+      }
```

Comment:
Should we log this case do you think?

Reply:
Further to our conversation on Slack just now, `idle_work_store.hpp` is scheduled for demolition in the near future.